lib/ansible/modules/cloud/openstack/os_network.py

```diff
@@ -196,10 +207,12 @@ def main():
 
                 if project_id is not None:
                     net = cloud.create_network(name, shared, admin_state_up,
-                                               external, provider, project_id)
+                                               external, provider, project_id,
+                                               port_security_enabled)
                 else:
                     net = cloud.create_network(name, shared, admin_state_up,
-                                               external, provider)
+                                               external, provider,
+                                               port_security_enabled)
```

Comment:
Same here, I suggest just adding `port_security_enabled=port_security_enabled` or something like that.  In this case, `project_id` is pointing to `port_security_enabled` value.

Reply:
done.